include/cantera/thermo/CoverageDependentSurfPhase.h

```diff
@@ -0,0 +1,476 @@
+ * dependency model, polynomial dependency model, and interpolative dependency model.
+ * In the dependency model equations, a coverage-dependent correction term is denoted
+ * by \f$ f^{cov} \f$ where \f$ f \f$ can be either enthalpy or entropy. Because
+ * lateral interaction can compose of both self- and cross- interactions, the total
+ * correction term of species \f$ k \f$ is a sum of all interacting species \f$ j \f$
+ * which can include itself. Coefficients \f$ c1_{k,j}-c7_{k,j} \f$ are user-provided
+ * parameters that can be given in input mechanism.
+ *
+ * Linear dependency model:
+ * \f[
+ *  f^{cov}_k(\theta) = \sum_j c1_{k,j} \theta_j
+ * \f]
+ *
+ * Piecewise-linear dependency model:
+ * \f[
+ * f^{cov}_k(\theta) = \sum_j \left\{
+ *  \begin{array}{ll}
+ *  c2_{k,j}\theta_j & \text{, } \theta_j \leq \theta^\text{change}_{k,j} \\
+ *  \left[c3_{k,j}(\theta_j - \theta^\text{change}_{k,j})
+ *  + (c2_{k,j}\theta^\text{change}_{k,j})\right]
+ *  & \text{, } \theta_j > \theta^\text{change}_{k,j} \\
+ *  \end{array}
+ *  \right.
+ * \f]
+ *
+ * Polynomial dependency model:
+ * \f[
+ *  f^{cov}_k(\theta) =
+ *   \sum_j \left[c4_{k,j}\theta_j + c5_{k,j}\theta_j^2
+ *           + c6_{k,j}\theta_j^3 + c7_{k,j}\theta_j^4\right]
+ * \f]
+ *
+ * Interpolative dependency model:
+ * \f[
+ *  f^{cov}_k(\theta) =
+ *   \sum_j \left[\frac{f^{cov}_k(\theta^{higher}_j) - f^{cov}_k(\theta^{lower}_j)}
+ *   {\theta^{higher}_j - \theta^{lower}_j}(\theta_j - \theta^{lower}_j)
+ *   + f^{cov}_k (\theta^{lower}_j)\right] \text{, }
+ *   \theta^{lower}_j \leq \theta_j < \theta^{higher}_j
+ * \f]
+ *
+ * Coverage-dependent heat capacity is calculated using an equation with a
+ * quadratic dependence on coverages and a logarithmic dependence on temperature.
+ * Temperature is nondimesionalized with a reference temperature of 1 K.
+ * The coverage-dependent heat capacity of species \f$ k \f$ is a sum of
+ * all quantities dependent on coverage of species \f$ j \f$. Coefficients
+ * \f$ c8_{k,j} \text{ and } c9_{k,j} \f$ are user-provided parameters that can be
+ * given in input mechanism.
+ *
+ * \f[
+ *  cp^{cov}_k(\theta) =
+ *   \sum_j \left(c8_{k,j} ln\left(\frac{T}{1\text{ K}}\right)
+ *   + c9_{k,j}\right) \theta_j^2
+ * \f]
+ */
+class CoverageDependentSurfPhase : public SurfPhase
+{
+public:
+    //! A struct to store sets of parameters used in coverage-dependent enthalpy
+    //! and entropy calculations by a polynomial equation or a linear equation
+    //! in CoverageDependentSurfPhase.
+    struct PolynomialDependency
+    {
+        /*!
+         * @param k index of a target species whose enthalpy and entropy are calculated
+         * @param j index of a species whose coverage affects enthalpy and entropy of
+         *          a target species
+         * @param enthalpy_coeffs array of polynomial coefficients describing
+         *                        coverage-depdendent enthalpy [J/kmol] in order of
```

Comment:
Here and in several other locations:
```suggestion
         *                        coverage-dependent enthalpy [J/kmol] in order of
```